src/main/java/no/nav/familie/inntektsmelding/integrasjoner/dokgen/RefusjonsendringPeriode.java

```diff
@@ -1,9 +1,12 @@
 package no.nav.familie.inntektsmelding.integrasjoner.dokgen;
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 
 import java.math.BigDecimal;
+import java.time.LocalDate;
+
 @JsonAutoDetect(getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, fieldVisibility = JsonAutoDetect.Visibility.ANY)
-public record RefusjonsendringPeriode(String fom, BigDecimal beloep) {
+public record RefusjonsendringPeriode(String fom, @JsonIgnore LocalDate fraDato, BigDecimal beloep) {
```

Comment:
Hvorfor trengs jsonignore her?

Reply:
Den trengs for å kunne sortere på fradato før listen av refusjonsendringer sendes fpdokgen

Reply:
den blir da ikke sendt med, brukes til å sortere på